### PR TITLE
Matplotlib

### PR DESCRIPTION
**Pull request by marikamurphy**
This is the completed matplotlib assignment

### CODE REVIEW

File: matplotlib/mpl_plotting/mpl_plotting.py
Comment (mm89039) [inline matplotlib/mpl_plotting/mpl_plotting.py]: I feel like there might be a better way to add labels to a correlation matrix

File: matplotlib/mpl_plotting/mpl_plotting.py
Comment (marikamurphy) [inline matplotlib/mpl_plotting/mpl_plotting.py:103]: I feel like there might be a more efficient way of adding labels to a correlation matrix.

Comment (timdiller) [reply matplotlib/mpl_plotting/mpl_plotting.py:103]: There is a package called `pandas` that makes this kind of thing much easier.  For now, this is fine, and the "pain" of creating the labels manually demonstrates the appeal of a package like Pandas.

File: matplotlib/mpl_plotting/mpl_plotting.py
Comment (timdiller) [inline matplotlib/mpl_plotting/mpl_plotting.py]: The only thing I don't like in your solution is the awkward double colorbar.  
screen shot 2018-02-09 at 4 10 21 pm
I think it can be eliminated by removing this line.  I'm pretty sure that `imshow` includes the colorbar by default, and maybe `matshow` doesn't.

**Review by timdiller — CHANGES_REQUESTED** [review]
One little nit pick.  Otherwise this looks good.  You're OK to merge after #1 has been merged.

**Review by timdiller — APPROVED** [review]

**Pull request merged**
